packages/manager/modules/billing/src/payment/method/add/index.html

```diff
@@ -85,10 +85,16 @@
                                     >
                                     </span>
                                 </p>
-                                <p
-                                    data-ng-switch-when="PAYPAL"
-                                    data-translate="billing_payment_method_add_selected_paypal_redirect"
-                                ></p>
+                                <p data-ng-switch-when="PAYPAL">
+                                    <span
+                                        data-ng-if="$ctrl.user.ovhSubsidiary === 'US'"
+                                        data-translate="billing_payment_method_add_selected_paypal_redirect_US"
+                                    ></span>
+                                    <span
+                                        data-ng-if="$ctrl.user.ovhSubsidiary !== 'US'"
+                                        data-translate="billing_payment_method_add_selected_paypal_redirect"
+                                    ></span>
+                                </p>
```

Comment:
`<p 
         data-ng-switch-when="PAYPAL" 
         data-ng-bind=":: ($ctrl.user.ovhSubsidiary === 'US' ? 'billing_payment_method_add_selected_paypal_redirect_US' : 'billing_payment_method_add_selected_paypal_redirect') | translate"></p>`

Reply:
Yes this works. Just to keep it simple and easy to understand, I have split it. At the end both offer same benefits.